blocksuite/affine/blocks/block-attachment/src/attachment-block.ts

```diff
@@ -190,13 +169,6 @@ export class AttachmentBlockComponent extends CaptionedBlockComponent<Attachment
         ${embedView
           ? html`<div class="affine-attachment-embed-container">
               ${embedView}
```

Comment:
By design or blocksuite, we should use `renderChildren` to render the child component. But We can keep it for now.